### PR TITLE
406 - document details

### PR DESCRIPTION
**Pull request by codyseibert**
screen shot 2018-12-31 at 5 32 08 pm

### CODE REVIEW

File: web-client/src/styles/custom.scss [hunk 1]
```diff
@@ -306,14 +304,77 @@ iframe {
...
+
+.toggles {
+  padding-right: 4px;
+  padding-left: 4px;
+}
+
+.toggle {
+  padding-top: 5px;
+  padding-bottom: 5px;
+  color: #0069a3;
```
Comment (kkoskelin) [inline web-client/src/styles/custom.scss]: do we have a USWDS color variable we can use here?

File: web-client/src/styles/custom.scss [hunk 2]
```diff
@@ -306,14 +304,77 @@ iframe {
...
+  cursor: pointer;
+  font-size: 13px;
+  text-align: center;
+  vertical-align: top;
+}
+
+.toggle:hover {
+  background-color: #0069a3;
```
Comment (kkoskelin) [inline web-client/src/styles/custom.scss]: color variables? (same for others)

File: web-client/src/tests/journey/docketClerkForwardWorkItem.js [hunk 3]
```diff
@@ -3,8 +3,10 @@ import _ from 'lodash';
 export default test => {
   return it('Docket clerk forward work item', async () => {
     test.setState('form', {
-      forwardRecipientId: 'seniorattorney',
-      forwardMessage: 'hello world',
+      [test.workItemId]: {
```
Comment (kkoskelin) [inline web-client/src/tests/journey/docketClerkForwardWorkItem.js:6]: does this construction make the value of test.workItemId the key?

File: web-client/src/views/DocumentDetail.jsx [hunk 4]
```diff
@@ -44,71 +47,138 @@ class DocumentDetail extends React.Component {
...
-                  <p>{document.filedBy}</p>
-                </div>
-              </div>
-              <span className="label" id="messages-label">
-                Messages
-              </span>
+            <div className="usa-width-one-third">
               {workItems.map((workItem, idx) => (
                 <div
-                  className="card messages-card"
```
Comment (kkoskelin) [inline web-client/src/views/DocumentDetail.jsx:65]: there was a class '.messages-card' in custom.scss.  is it also removed?

File: web-client/src/views/DocumentDetail.jsx [hunk 5]
```diff
@@ -44,71 +47,138 @@ class DocumentDetail extends React.Component {
...
-                    {showForwardInputs && (
+
+                  <div className="usa-grid-full extra pt-1 pb-1 toggles">
+                    <div
+                      className={`usa-width-one-third toggle ${
+                        showAction('history', workItem.workItemId)
+                          ? 'selected'
+                          : ''
+                      }`}
+                      onClick={() =>
```
Comment (kkoskelin) [inline web-client/src/views/DocumentDetail.jsx:101]: Elements with click handlers ought to be either A or BUTTON elements, something that is natively focusable with keyboard. In this case, it probably makes sense that they are converted to BUTTON elements, but nevertheless styles as you have them now. (some css fidgeting required I'm certain)

**Review by kkoskelin — CHANGES_REQUESTED** [review]

File: web-client/src/tests/journey/docketClerkForwardWorkItem.js (hunk 3 above)
Comment (codyseibert) [reply web-client/src/tests/journey/docketClerkForwardWorkItem.js:6]: it seems to do the trick

Comment (kkoskelin) [reply web-client/src/tests/journey/docketClerkForwardWorkItem.js:6]: I'll take that as a 'yes'. :)

Comment (ericsorenson) [reply web-client/src/tests/journey/docketClerkForwardWorkItem.js:6]: 👍

File: web-client/src/views/DocumentDetail.jsx (hunk 4 above)
Comment (ericsorenson) [reply web-client/src/views/DocumentDetail.jsx:65]: it's been removed

File: web-client/src/styles/custom.scss (hunk 2 above)
Comment (ericsorenson) [reply web-client/src/styles/custom.scss]: Updated!

File: web-client/src/styles/custom.scss (hunk 1 above)
Comment (ericsorenson) [reply web-client/src/styles/custom.scss]: Updated!

File: web-client/src/views/DocumentDetail.jsx [hunk 6]
```diff
@@ -44,71 +47,138 @@ class DocumentDetail extends React.Component {
...
+                    >
+                      <FontAwesomeIcon icon="list-ul" size="sm" /> View History
+                    </button>
+                    <div
+                      className={`usa-width-one-third toggle ${
+                        showAction('complete', workItem.workItemId)
+                          ? 'selected'
+                          : ''
+                      }`}
+                      onClick={() =>
```
Comment (kkoskelin) [inline web-client/src/views/DocumentDetail.jsx:116]: There's several more DIVs here with onclick events which should be buttons FYI.

Comment (ericsorenson) [reply web-client/src/views/DocumentDetail.jsx:116]: Updated!

File: web-client/src/views/DocumentDetail.jsx (hunk 5 above)
Comment (ericsorenson) [reply web-client/src/views/DocumentDetail.jsx:101]: Updated!

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**